Formula/fastd.rb

```diff
@@ -26,10 +26,10 @@ class Fastd < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "libuecc"
-  depends_on "libsodium"
   depends_on "bison" => :build # fastd requires bison >= 2.5
   depends_on "pkg-config" => :build
+  depends_on "libuecc"
```

Comment:
put them back :)

Reply:
I thought you were slowly fixing these? Is why I moved them. Maybe I just imagined that though 🙈.

Reply:
only when the line is changing for some other reason (e.g. depends_on "ghc@8.0" -> depends_on "ghc" is an excuse to move it below depends_on "cabal")